### PR TITLE
Update OrthoVSPerspective.py

### PR DESCRIPTION
**Pull request by Prabhnith**
little errors in the documentation and this example was not working.

### CODE REVIEW

File: examples.py/3D/Camera/OrthoVSPerspective.py [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
+function specify the clipping volume where left and right
+are the minimum and maximum x values respectively, 
+top and bottom are the minimum and maximum y values respectively, 
+and lastly, the near and far are the minimum and maximum z values.
  """
+showPerspective = True
 def setup():
     size(640, 360, P3D)
-    noStroke()
+    noFill()
```
Comment (GoToLoop) [inline examples.py/3D/Camera/OrthoVSPerspective.py]: Why add **noFill()** while the next line is **fill()**?!

File: examples.py/3D/Camera/OrthoVSPerspective.py [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
-
-    if mousePressed:
-        fov = PI / 3.0
-        cameraZ = (height / 2.0) / tan(PI * fov / 360.0)
-        perspective(fov, float(width) / float(height),
-                                cameraZ / 2.0, cameraZ * 2.0)
+    background(0)
+    far = map(mouseX, 0, width, 120, 400)
+    if showPerspective:
+        perspective(PI / 3.0, float(width) / float(height), 10, far)
```
Comment (GoToLoop) [inline examples.py/3D/Camera/OrthoVSPerspective.py]: Those **float()** conversions are redundant b/c upcasting from integral to fractional type is automatic.

Comment (GoToLoop) [inline examples.py/3D/Camera/OrthoVSPerspective.py]: Processing got constant THIRD_PI, which corresponds to `PI / 3.0`.  :wink:

File: examples.py/3D/Camera/OrthoVSPerspective.py [hunk 3]
```diff
@@ -1,36 +1,41 @@
...
     else:
-        ortho(-width / 2, width / 2, -height / 2, height / 2, -10, 10)
-
-
+        ortho(0, width, 0, height, 10, far)
+ 
     translate(width / 2, height / 2, 0)
     rotateX(-PI / 6)
     rotateY(PI / 3)
```
Comment (GoToLoop) [inline examples.py/3D/Camera/OrthoVSPerspective.py]: Can be replaced by rotateY(THIRD_PI)

File: examples.py/3D/Camera/OrthoVSPerspective.py [hunk 4]
```diff
@@ -1,36 +1,41 @@
...
+    if showPerspective:
+        perspective(PI / 3.0, float(width) / float(height), 10, far)
     else:
-        ortho(-width / 2, width / 2, -height / 2, height / 2, -10, 10)
-
-
+        ortho(0, width, 0, height, 10, far)
+ 
     translate(width / 2, height / 2, 0)
     rotateX(-PI / 6)
```
Comment (GoToLoop) [inline examples.py/3D/Camera/OrthoVSPerspective.py]: Can be replaced by `rotateY(-.5 * THIRD_PI)`

**Comment by jdf** [comment]
Thank you.

**Pull request merged**